Packages/com.unity.inputsystem/InputSystem/Plugins/Android/AndroidSensors.cs

```diff
@@ -276,6 +279,48 @@ public class AndroidGameRotationVector : AttitudeSensor
     public class AndroidStepCounter : StepCounter
     {
     }
+
+    /// <summary>
+    /// Hinge angle sensor device on Android.
+    /// This sensor is usually available on foldable devices.
+    /// </summary>
+    /// <seealso href="https://developer.android.com/reference/android/hardware/Sensor#TYPE_HINGE_ANGLE/>
+    [InputControlLayout(stateType = typeof(AndroidSensorState), variants = nameof(AndroidSensorType.HingeAngle), hideInUI = true)]
+    public class AndroidHingeAngle : Sensor
```

Comment:
My main concern about this PR is that it adds public platform-specific API surface with no platform-specific particularities? Wouldn't this be applicable for any platform if mapped to existing HID sensors out there also on other platforms? Hinge-sensors have been around for a long time on e.g. laptops. 

It's called out in HID Usages standard even though often vendor denied: https://www.usb.org/sites/default/files/hut1_4.pdf, see "23.20.1 Hinge Sensors". Hence I believe there is value to standardise the interface we want for hinge sensor and add it as a non-platform specific sensor type instead if we expand sensor support? I cannot see that this scales. What would happen to this class if Apple adds it or there is iOS or Microsoft APIs added for laptops in your perspective?

Reply:
So what you're basically suggesting is to have a HingeAngle class and AndroidHingeAngle derive from it ?

I can do that of course, didn't consider laptops.

Reply:
https://github.com/Unity-Technologies/InputSystem/pull/1979/commits/4fbd7fd49cfbb510422e2de643e6abf09596a3a2

Reply:
Thanks for doing the update, it seem to make more sense to provide a non-platform specific interface for a sensor as this.